crates/iota-core/src/execution_cache/writeback_cache.rs

```diff
@@ -2232,6 +2232,12 @@ impl StateSyncAPI for WritebackCache {
         transaction: &VerifiedTransaction,
         transaction_effects: &TransactionEffects,
     ) -> IotaResult {
+        self.store
+            .insert_transaction_and_effects(transaction, transaction_effects)
+            .map_err(IotaError::from)?;
```

Comment:
Do we need `.map_err(IotaError::from)` there? IIRC, the `?` operator should do that already.